app/server/locale/ussd.en.yml

```diff
@@ -75,20 +75,20 @@ en:
         10. Show previous
     account_management: |
       CON My account
-      1. My Business
+      1. My Profile
       2. Change language
       3. Check Balances
       4. Change PIN
       5. Opt Out of Market Place
       0. Back
-    my_business: |
-      CON My Business
-      1. See my business
-      2. Change my business
-      0. Back
-    about_my_business:
-      bio: END You sell %{user_bio}
-      none: END No business set, please go to Change My Business to set it to something
+    about_me:
+      profile: |
+        END You details are:
+            Name %{full_name}
+            Gender %{gender}
+            Location %{location}
+            You sell %{user_bio}
+      none: END Some profile information is not set, please go to My profile to set %{empty_attrs}.
```

Comment:
`none` means _no_ profile information not set?

Reply:
It means some profile information is missing.

Reply:
@WillRuddick  the changes to the yml files are on this PR